packages/client/hmi-client/src/page/Home.vue

```diff
@@ -28,6 +34,12 @@ const selectedDocument = ref<DocumentType>();
 
 const resourcesStore = useResourcesStore();
 const queryStore = useQueryStore();
+const router = useRouter();
+const auth = useAuthStore();
+
+const isNewProjectModelVisible = ref(false);
```

Comment:
nitpick: Did you mean?
- newProjectModal, or
- newProjectMode

Not sure if there is a typo here or not.

Reply:
Good catch, should be `newProjectModalVisible`